ac97forvirtualbox.txt

```diff
@@ -1,88 +1,88 @@
-Description = Estrarre nella cartella "ReactOS" e poi riavviare ReactOS due volte.
-
-[Section.040c]
-Name = Pilote AC97 pour VirtualBox
-License = Inconnue
-Description = Décompresser dans le dossier "ReactOS" puis redémarrer ReactOS deux fois.
-Size = 185,5 Kio
-
-[Section.0413]
-Name = AC97 Driver voor VirtualBox
-License = Ongekend
-Description = Pak uit in de "ReactOS" map en start tweemaal opnieuw op.
-
-[Section.0415]
-Name = Sterownik AC97 dla VirtualBox
-License = Nieznana
-Description = Rozpakuj zawartość w folderze "ReactOS" i dwukrotnie zrestartuj system.
-
-[Section.0418]
-Name = Modul pilot AC97 pentru VirtualBox
-License = Nespecificată
-Description = Dezarhivați în dosarul ReactOS apoi reporniți ReactOS de două ori.
-Size = 185,5 Kio
-
-[Section.0419]
-Name = Драйвер AC97 для VirtualBox
-License = Не указано
-Description = Pазархивируйте содержимое в папку "ReactOS", затем дважды перезагрузите систему.
-Size = 185,5 КиБ
-
-[Section.041f]
-Name = VirtualBox için AC'97 Sürücüsü
-License = Bilinmiyor
-Description = AC'97 uyumlu aygıtlar için ses sürücüsü. "ReactOS" dizinine çıkartınız ve bilgisayarı iki kez yeniden başlatınız.
-Size = 185,5 KiB
-
-[Section.0422]
-Name = Драйвер AC97 для VirtualBox
-License = Невідома
-Description = Pозархівуйте вміст в теку "ReactOS" після чого двічі перезавантажте систему.
-
-[Section.0425]
-Name = AC'97 draiver VirtualBox'ile
-License = Tundmatu
-Description = Audio driver for AC'97-compatible devices. Unzip in the ReactOS folder and restart twice.
-Size = 185,5 KiB
-
-[Section.0804]
-Name = 适用于 VirtualBox 的 AC'97 驱动程序
-License = 未知
-Description = AC'97 兼容设备的音频驱动程序。解压到 ReactOS 文件夹并重启两次后生效。
-
-[Section.0813]
-Name = AC97 Driver voor VirtualBox
-License = Ongekend
-Description = Pak uit in de "ReactOS" map en start tweemaal opnieuw op.
-
+[Section]
+Name = AC'97 Driver for VirtualBox
+Version = 5.10.00.3610
+License = Unknown
+Description = Audio driver for AC'97-compatible devices. Unzip in the ReactOS folder and restart twice.
+Size = 185.5 KiB
+Category = 13
+URLDownload = http://svn.reactos.org/packages/ac97_vbox.exe
+SHA1 = 050b74c2f1b7aacaf2c2ff5068b89a24e4495d21
+CDPath = none
+SizeBytes = 189954
+
+[Section.0405]
+Name = Ovladač AC97 pro VirtualBox
+License = Neznámá
+Description = Rozbalte do složky "ReactOS" a pak ReactOS dvakrát restartujte.
+
+[Section.0407]
+Name = AC97 Treiber für VirtualBox
+License = Unbekannt
+Description = Sound Treiber für AC'97-kompatible Geräte. Entpacken in das "ReactOS"-Verzeichnis und ReactOS zweimal neustarten.
+Size = 185,5 KiB
+
+[Section.0a]
+Name = Controlador AC'97 para VirtualBox
+License = Desconocida
+Description = Controlador de audio para dispositivos compatibles con AC'97. Se debe descomprimir en la carpeta «ReactOS» y reiniciar dos veces.
+
+[Section.10]
+Name = Driver AC97 per VirtualBox
+License = Sconosciuta
+Description = Driver audio per dispositivi compatibili con AC'97. Decomprimere nella cartella ReactOS e riavviare due volte.

```

Comment:
I think should be maintained as "ReactOS" (with quotes) ora << ReactOS>> like in spanish translation.